src/app/page.tsx

```diff
@@ -1,8 +1,163 @@
 import Image from "next/image";
+import { Jaldi } from "next/font/google";
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook } from '@fortawesome/free-brands-svg-icons';
+import { faInstagram } from "@fortawesome/free-brands-svg-icons";
+import { faYoutube } from "@fortawesome/free-brands-svg-icons";
+import { faLinkedin } from "@fortawesome/free-brands-svg-icons";
+
 
 export default function Home() {
   return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
+    <div className="flex flex-col min-h-screen">
+      <link
+        href="https://fonts.googleapis.com/css2?family=Jaldi:wght@400;700&display=swap"
+        rel="stylesheet"
+      />
+      
+      <header className="sticky top-0 bg-[#971B2F] shadow-lg z-50" >
+        <div className = "max-w-screen-xl mx-auto px-2 py-3 flex items-center">
+          <div className="flex items-center transform -translate-x-20 space-x-5">
+            <img
+              src = "/logo2.png"
+              alt = "Umass Photo logo"
+              width = {60}
+              height = {60}
+            />
+            <p className= "font-Jaldi text-white text-3xl font-bold">Umass Photo</p>
+          </div>
+
+          <div className="ml-auto flex items-center pr-7">
+            <a className = "font-Jaldi text-white text-2xl font-normal mr-16" href = "#">home</a>
+            <a className = "font-Jaldi text-white text-2xl font-normal mr-16" href = "#">gallery</a>
+            <a className = "font-Jaldi text-white text-2xl font-normal mr-16" href = "#">events</a>
+            <a className = "font-Jaldi text-white text-2xl font-normal mr-16" href = "#">about</a>
+            <a className = "font-Jaldi text-white text-2xl font-normal ml-auto" href = "#">contact</a>
+          </div>
+        </div>
+      </header>
+
+      <main className="flex-grow">
```

Comment:
The background becomes black when the browser enables dark mode (see screenshot).
You could fix this by explicitly setting the background to be white, or you could have the text also respond to dark mode.
```suggestion
      <main className="flex-grow bg-white">
```
Screenshot:
![image](https://github.com/user-attachments/assets/11e84406-05ae-482f-9dbb-8e1864a1ff5d)